packages/kubernetes-1.29/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+# "." is not allowed in crate names, but we want a friendlier name for the
+# directory and spec file, so we override it below.
+name = "kubernetes-1_29"
+version = "0.1.0"
+edition = "2021"
+publish = false
+build = "../build.rs"
+
+[lib]
+path = "../packages.rs"
+
+[package.metadata.build-package]
+package-name = "kubernetes-1.29"
+
+[[package.metadata.build-package.external-files]]
+url = "https://distro.eks.amazonaws.com/kubernetes-1-29/releases/1/artifacts/kubernetes/v1.29.0/kubernetes-src.tar.gz"
+sha512 = "fb94f967683428ca59dd8f203f398b2658ef5b65d517672d16e7f5b4e08662f9ee81065840060301e240f7659ec61543402ad4d0379c5405047c612ede8ad4eb"
+
+# RPM BuildRequires
+[build-dependencies]
+glibc = { path = "../glibc" }
+
+# RPM Requires
+[dependencies]
+aws-signing-helper = { path = "../aws-signing-helper" }
+ecr-credential-provider-1_27 = { path = "../ecr-credential-provider-1.27" }
```

Comment:
Might want to check v1.29 of the ECR credentials provider and see if we need it or if we can stay on v1.27.

Should also consider maybe packaging the ecr credential provider together with the k8s package.

Reply:
I'll add v1.29 of the ECR credentials provider package and test it.